ios/FluentUI/TwoLineTitleView/TwoLineTitleView.swift

```diff
@@ -83,53 +101,40 @@ open class TwoLineTitleView: UIView, TokenizedControlInternal {
         case disclosure
         case downArrow
 
-        var image: UIImage? {
+        public func image(forSize size: GlobalTokens.IconSizeToken) -> UIImage? {
             let image: UIImage?
-            switch self {
-            case .disclosure:
-                image = UIImage.staticImageNamed("chevron-right-20x20")
-            case .downArrow:
-                image = UIImage.staticImageNamed("chevron-down-20x20")
-            case .none:
+            switch (self, size) {
+            case (.disclosure, .size120):
```

Comment:
Can you combine the cases that return that same images?

Reply:
Ah, never mind. They're all different images, I got tricked by the sizes.

Reply:
But, considering how many cases there are in what's more or less the default case, I'm not sure a switch is the best choice here.

Reply:
See this comment. https://github.com/amgleitman/fluentui-apple/pull/1#discussion_r1190462597